qa/L0_warmup/test.sh

```diff
@@ -143,7 +150,12 @@ for BACKEND in ${BACKENDS}; do
         echo -e "\n***\n*** Failed. Expected warmup for stateless model\n***"
         RET=1
     fi
-    grep "is running warmup sample 'sequence sample'" $SERVER_LOG
+    grep "is running warmup sample 'sequence sample' for iteration 1" $SERVER_LOG
+    if [ $? -ne 0 ]; then
+        echo -e "\n***\n*** Failed. Expected warmup for stateful model\n***"
+        RET=1
+    fi
+    grep "is running warmup sample 'sequence sample' for iteration 2" $SERVER_LOG
```

Comment:
Perhaps add which iteration failed in the echoed statement for quicker debugging.